docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -149,23 +155,27 @@ To add a Semgrep configuration snippet in your GitLab CI/CD pipeline:
     defaultValue="glcicd-semgrep"
     values={[
     {label: 'Default', value: 'glcicd-semgrep'},
-    {label: 'Semgrep Supply Chain', value: 'glcicd-ssc'},
+    {label: 'Semgrep OSS', value: 'glcicd-oss'},
     ]}
 >
 
 <TabItem value='glcicd-semgrep'>
 
-The following configuration creates a CI job that runs a SAST and SCA scan using both Semgrep Code and Semgrep Supply Chain.
+The following configuration creates a CI job that runs scans depending on what products you have enabled in Semgrep Cloud Platform.
 
 <GlcicdSemgrepAppSast />
```

Comment:
I think this is the only sample that's in a collapsible box, and it might not need to be?

Reply:
I think the tabs here were broken. Fixed.